tasks/eth/ink-002-set-respected-game-type/.env

```diff
@@ -0,0 +1,4 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+#`OWNER_SAFE` represents the address of the `deputyGuardian` gnosis safe attached to the `DeputyGuardianModule`
+OWNER_SAFE=0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A # guardian multisig
+SAFE_NONCE=
```

Comment:
Should be 99 per our notion doc

Reply:
updated to 98, as shown in the notion's pages new ordering